syntax = "proto3";

package internalAgreement;

message UpstreamACK
{
	uint32 MsgID = 1;		// 消息号
	
	bytes Data = 2;		// 用户封包
	
	
	int64 ClientID = 3;	// 玩家在router的id
}


message CloseClientACK
{
	int64 ClientID = 1;
}

message DownstreamACK
{
	uint32 MsgID = 1;		// 消息号
	
	bytes Data = 2;		// 用户封包
	
	
	repeated int64 ClientID = 3;	// 要广播的玩家列表
}

message LogicServerLogin
{
    int32 id = 1;
    bool isPrimary = 2;
}

message LogicServerLoginReply
{
    bool isSuccess = 1;
    int32 id = 2;
}


message LogicServerKickPlayer
{
    int64 RoleRuntimeID = 1;
}

message LogicServerSetRoleSlave
{
    int64 RoleRuntimeID = 1;
    bool willSet = 2;
}
